usefulUtilities/loudApp/appResources/loudApp_app.js

```diff
@@ -0,0 +1,559 @@
+        activeTargetUUID = targetUUID;
+    }
+
+
+    // Stops soloing target avatar
+    function stopListening() {
+        if (DEBUG) {
+            console.log("stopListening activeTargetUUID: " + activeTargetUUID + "userStore[activeTargetUUID]" + JSON.stringify(userStore[activeTargetUUID]));
+        }
+        Audio.resetSoloList();
+        if (userStore[activeTargetUUID]) {
+            userStore[activeTargetUUID].isToggled = false;
+        }
+        settings.ui.isListening = false;
+        activeTargetUUID = null;
+    }
+
+    // #endregion AUDIO
+
+
+    // #region LIST MANAGEMENT
+
+    // return all avatars in radius
+    function getAvatarsInRadius(radius) {
+        return AvatarList.getAvatarsInRange(MyAvatar.position, radius).filter(function (uuid) {
+            return uuid !== MyAvatar.sessionUUID;
+        });
+    }
+
+
+    // searches the top 10 list to get the index
+    function getIndexOfSettingsUser(uuid) {
+        if (settings.users.length) {
+            var index = settings.users.map(function (item) {
+                // print(item.uuid)
+                return item.uuid;
+            }).indexOf(uuid);
+            return index;
+        }
+        return -1;
+    }
+
+    // #endregion LIST MANAGEMENT
+
+
+    // #region APP
+
+    // Create the tablet button and setup app callbacks
+    var ui,
+        BUTTON_NAME = "SEEK LOUD",
+        APP_URL = Script.resolvePath('./resources/loudApp_ui.html'),
+        EVENT_BRIDGE_OPEN_MESSAGE = "eventBridgeOpen";
+    function startup() {
+        // Create the tablet app
+        ui = new AppUi({
+            buttonName: BUTTON_NAME,
+            onMessage: onMessage, // UI event listener  
+            home: APP_URL,
+            graphicsDirectory: Script.resolvePath("./resources/icons/"),
+            onOpened: onOpened,
+            onClosed: onClosed
+        });
+
+        AvatarList.avatarAddedEvent.connect(addUser);
+        AvatarList.avatarRemovedEvent.connect(removeUser);
+        Users.usernameFromIDReply.connect(setUserName);
+
+        Script.scriptEnding.connect(unload);
+
+        startUpdateInterval();
+    }
+
+
+    // Open the app callback
+    function onOpened() {
+        Users.requestsDomainListData = true;
+        startUpdateInterval();
+        addAllOverlays();
+    }
+
+
+    // Close the app callback
+    function onClosed() {
+        Users.requestsDomainListData = false;
+        stopUpdateInterval();
+        removeAllOverlays();
+    }
+
+
+    // Update UI
+    function doUIUpdate(key) {
+        ui.sendToHtml({
+            type: UPDATE_UI,
+            value: key ? settings[key] : settings,
+            key: key ? key : ""
+        });
+    }
+
+
+    // Sorts users by loudness in userStore then populates ten loudest list
+    var SEARCH_RADIUS_DEFAULT = 10; // radius that avatars are factored in muting and top 10 loudest
+    function sortAvatarsByLoudness() {
+        // sort by audioLevel
+        function sortNumber(a, b) {
```

Comment:
same comment about lint

Reply:
I think my linter didn't survive the computer shuffle. I'll reconfigure it.